docs/detections/api/rules/rules-api-update.asciidoc

```diff
@@ -166,6 +167,20 @@ uses both `and` and `or` logic.
 
 |==============================================
 
+===== Fields required for new terms rules `PUT` calls
+
+[width="100%",options="header"]
+|==============================================
+|Name |Type |Description
+
+|new_terms_fields |String[] |Fields to monitor for new values. Must contain 1-3 field names.
+
+|history_window_start |String |Start date to use when checking if a term has been seen before.
+Supports relative dates, e.g. `now-30d` will search the last 30 days of data when checking if a term
+is new.
```

Comment:
[Same comment as for `rules-api-update`](https://github.com/elastic/security-docs/pull/3914/files#r1322987743).